package.json

```diff
@@ -81,6 +81,7 @@
     "loglevelnext": "^3.0.1",
     "marked": "0.7.0",
     "mocha": "6.2.2",
+    "napa": "^3.0.0",
```

Comment:
why add napa, it seems to not be required for elm-pep?

Reply:
Right, that's an accidental leftover from a previous attempt trying to integrate https://github.com/mpizenberg/elm-pep directly from github, which does not have a package.json. Removed.